compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -1798,6 +1819,39 @@ private void handleErrorRelatedBuiltInFunctions(BLangInvocation iExpr, BLangBuil
         }
     }
 
+    private void handleBuiltInFunctions(BLangInvocation iExpr, BLangBuiltInMethod function) {
+        if (iExpr.argExprs.size() > 0) {
+            dlog.error(iExpr.pos, DiagnosticCode.TOO_MANY_ARGS_FUNC_CALL, function.getName());
+        }
+        resultType = this.symTable.getTypeFromTag(TypeTags.BOOLEAN);
+    }
+
+    private void checkBuiltinLengthInvocationExpr(BLangInvocation iExpr) {
+        if (iExpr.argExprs.size() > 0) {
+            dlog.error(iExpr.pos, DiagnosticCode.TOO_MANY_ARGS_FUNC_CALL, "length");
+            return;
+        }
+        iExpr.type = this.symTable.getTypeFromTag(TypeTags.INT);
```

Comment:
`symTable.booleanType`, `symTable.intType` can be used instead of `symTable.getTypeFromTag(...)` ?

Reply:
Changed with d548812